File: Python/Project/Activity4.py
from selenium.webdriver.chrome import webdriver
from selenium.webdriver.support.wait import WebDriverWait
from selenium import webdriver
from selenium.webdriver.common.by import By

driver = webdriver.Chrome()
wait = WebDriverWait(driver, 10)


def openURL():
    options = webdriver.ChromeOptions()
    options.add_experimental_option("detach", True)
   
    driver.get('https://alchemy.hguy.co/lms')


def verifySecondMostPopularCourseTitle():
    expectedTitle = 'Email Marketing Strategies'
    actualTitle = driver.find_element(By.XPATH,
                                      "//h3[@class='entry-title' and text()='Email Marketing Strategies']").text
    print('The most popular Course Title is:', actualTitle)
    assert expectedTitle == actualTitle
    print('Assertion completed successfully!')


def browserClose():
    driver.quit()

# Open the URL
openURL()

# Verify the second most popular course
verifySecondMostPopularCourseTitle()

# Close the browser
browserClose()

File: Selenium/Project/Activity4.java
package SeleniumLMS;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;

public class Activity4 {
    public static void main(String[]args)
    {
        WebDriverManager.firefoxdriver().setup();

        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE,"NULL");

        WebDriver driver = new FirefoxDriver();

        driver.get("https://alchemy.hguy.co/lms");

        String secondCourse = driver.findElement(By.xpath("//*[contains(text(),'Email Marketing Strategies')]")).getText();

        System.out.println(secondCourse);

        Assert.assertEquals("Email Marketing Strategies",secondCourse);

        if(secondCourse.contains("Email Marketing Strategies"))
        {
            driver.close();
        }

        else
        {
            System.out.println("Landed of wrong course");

        }
    }
}
